ONSdigital/eq-author-app
# EAR-1586-fix import of History objects

https://collaborate2.ons.gov.uk/jira/browse/EAR-1586

### What is the context of this PR?

This ticket is to deal with the broken History page after an import of a survey

**The issue**
Migrate a survey from AWS to GCP or,
_Duplicate in GCP a survey migrated from AWS _
Enter the survey and, either try to view the survey (using the view survey button) or, the history
History page is breaking, showing oops, something went wrong!

### How to review

- Use the import bookmarklet (https://github.com/ONSdigital/eq-author-app/wiki/import-survey-bookmarklet )
-  to import a survey from AWS Prod ( https://prod-author.prod.eq.ons.digital/#/ ) to your local version of this PR
- (use https://prod-author.prod.eq.ons.digital/export/ relevant questionnaireID
- Go to you local version of Eq-author
- Hit the import bookmarklet link
- paste the JSON into relevant place on the import tool and import
- refresh the page and it should take you to the imported questionnaire on your local

Check that the history page is working and showing no "broken" messages
check you are then still able to add and delete history notes
Check you can "View Survey"
